pkg/util/pod/pod.go

```diff
@@ -43,6 +45,27 @@ func DefineWithHostNetwork() *corev1.Pod {
 	return podObject
 }
 
+// RedefineWithNodeSelector - Update pod definition with node selector
```

Comment:
Can you rephrase the comment in a more imperative way? 
Like `RedefineWithNodeSelector updates the pod definition with..". 
Same as per comments below

Reply:
Done